sidebars.js

```diff
@@ -152,6 +152,7 @@ module.exports = {
       ],
     },
     'external-resources',
+    'cadence-to-temporal',
```

Comment:
personally think we should be very sparing with top level menu items. keep in mind that most people will not be migrating so this will be irrelevant to them.  what do you think of sticking it under Server?

Reply:
Info in here is not Server specific.
Part of the objective is to increase discoverability, and this info already exists as part of the changelog. Burying it only defeats the purpose.
Its existence also has a time-limit. So its there for now while we need it - then it goes away.

Reply:
Maybe we could have an "Additional resources" ?

Reply:
ah if it is time limited then thats not such a big deal :)

i dislike "additional resource" because too vague. in absence of a better idea, lets keep this as-is!